### DISCUSSION TITLE
Why is Nomacs making network requests to localhost?

### DISCUSSION THREAD

**Discussion by e-t-l** [nomacs/nomacs#1177] [Q&A]
I noticed my firewall logged dozens to hundreds of requests from Nomacs to 127.0.0.1 on various ports in the 40000+ range. What is the purpose of these connections? (it also made a single external network request to nomacs.org, presumably to check for updates. I am NOT asking about this one.)

**Comment by scrubbbbs** [comment]
Nomacs is looking for other instances of nomacs for the sync feature (sync menu in the main menu)
